### ISSUE TITLE
Are there any plans to support .NET Core?

### ISSUE THREAD

**Issue by TAGC** [giacomelli/DocsByReflection#6]
Currently this library only targets the .NET Framework. It would be nice if it supported .NET Core as well, since most code is heading that way.

**Comment by giacomelli** [comment]
No plan at the moment, but is something we'll  need to do in a near future.

**Comment by wonea** [comment]
Agreed, I've just found this library and would love a NETStandard version.

**Comment by wonea** [comment]
My DTO class is in NETSTANDARD and I get the following error when calling from NET Framework 4.7

![image](https://user-images.githubusercontent.com/557192/31502338-ad8e01de-af64-11e7-87f5-145761ed4d59.png)

**Comment by pablopioli** [comment]
Trying to debug a bug I copied the source code to a netstandard 2 project.
It worked without changes, so it's doable.

**Comment by onionhammer** [comment]
It would be great if this project supported .net standard
